### ISSUE TITLE
Windows node agent can't connect to Datadog cluster agent on EKS cluster [BUG] 

### ISSUE THREAD

**Issue by julianll1202** [DataDog/datadog-agent#28904]
**Agent Environment**

Output of agent status
```
 Loading Errors
  ==============
    orchestrator_pod
    ----------------
      Core Check Loader:
        Could not configure check orchestrator_pod: orchestrator check is configured but the cluster name is empty

      Python Check Loader:
        unable to import module 'orchestrator_pod': No module named 'orchestrator_pod'
```
**Describe what happened:**
Windows node agent can't connect to Datadog cluster agent on EKS cluster.

**Describe what you expected:**
Windows node agent connecting to datadog cluster agent and windows pods appearing on datadog's website.

**Steps to reproduce the issue:**
At the moment I do not know how to reproduce the issue.

**Additional environment details (Operating System, Cloud provider, etc):**
* **Agent version** 7.56.0
* **Helm chart version** 3.70.3
* **Kubernetes version** 1.29
* **Provider** AWS
* **Cluster type** EKS cluster

**Comment by mfpierre** [comment]
Hi @julianll1202 it seems there could be 2 different issues:
- from the output of the status it seems that the agent is not able to detect automatically the cluster name in your cluster, it's mandatory for the orchestrator check to be able to run, you can set it via the [chart](https://github.com/DataDog/helm-charts/blob/3097984ae336c57ada9a41b2e7a9032c58b99440/charts/datadog/values.yaml#L96) or the `DD_CLUSTER_NAME` env variable 
- the issue title is about cluster agent <> node agent communication, do you have more info about that?

**Comment by julianll1202** [comment]
Yes, I have a mixed cluster (Linux & Windows) and I want my Windows Datadog agent to connect to the Datadog Cluster Agent, which can only be Linux so that I can monitor them

**Comment by jacjtito** [comment]
Hi @julianll1202 👋 — thanks again for taking the time to report this issue.

We're closing this due to inactivity: it's been over 6 months since the last update, and the stale bot's check-in didn't receive a response. This isn't a reflection of the value of the contribution — it's purely a hygiene pass to keep the backlog actionable.

If this is still relevant to you:
- Feel free to **reopen this issue** or open a new one with updated reproduction steps
- If the underlying issue is still present, a fresh report against the current version will get faster attention

Thanks for your patience and for contributing to the Datadog Agent! 🙏
